packages/app/pages/Inventory/inventory.scss

```diff
@@ -111,3 +129,13 @@
          }
      }
 }
+//
```

Comment:
Does this need to be here?

Reply:
Nope! I removed it and the commented out CSS underneath. That was left over from an earlier styling attempt.